src/ripple/app/tx/impl/XChainBridge.cpp

```diff
@@ -277,11 +303,11 @@ enum class OnTransferFail {
 
 struct FinalizeClaimHelperResult
```

Comment:
Maybe move `struct FinalizeClaimHelperResult` up? So that `finalizeClaimHelper()` and its function document are together.

Reply:
Fixed in e040076e5a `[fold] Move return helper structs so they are above the function docstrings`